mono/metadata/sgen-mono.c

```diff
@@ -2639,7 +2639,7 @@ mono_gc_get_allocated_bytes_for_current_thread (void)
 	info = mono_thread_info_current ();
 
 	/*There are some more allocated bytes in the current tlab that have not been recorded yet */
-	return info->total_bytes_allocated + info->tlab_next - info->tlab_start;
+	return info->total_bytes_allocated + (ptrdiff_t)(info->tlab_next - info->tlab_start);
```

Comment:
This cast is not needed. Subtracting two pointers already returns `ptrdiff_t`.

Reply:
This entire change does not do anything anyway.
You have integer + pointer - pointer
which isn't going to lose any bits or return a different type than it is already doing.